src/components/menu/menu.spec.tsx

```diff
@@ -280,13 +247,17 @@ describe("Menu", () => {
         openSubmenu(wrapper, 0);
 
         act(() => {
-          menuItem.props().onClick();
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          menuItem.props().onClick!({} as React.MouseEvent<HTMLButtonElement>);
         });
 
         wrapper.update();
 
         act(() => {
-          menuItem.props().onKeyDown(events[key]);
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          menuItem.props().onKeyDown!(
+            (events[key] as unknown) as React.KeyboardEvent<HTMLButtonElement>
+          );
```

Comment:
Similar to the previous comment
```suggestion
          menuItem.simulate("keydown", { key: events[key] });
```